localstack/services/ssm/provider.py

```diff
@@ -21,11 +22,102 @@
 )
 from localstack.services.moto import call_moto, call_moto_with_request
 from localstack.utils.aws import aws_stack
+from localstack.utils.collections import remove_attributes
+from localstack.utils.objects import keys_to_lower
 
 PARAM_PREFIX_SECRETSMANAGER = "/aws/reference/secretsmanager"
 
 
+class ValidationException(CommonServiceException):
+    def __init__(self, message=None):
+        super().__init__("ValidationException", message=message, sender_fault=True)
+
+
+# TODO: check if _normalize_name(..) calls are still required here
 class SsmProvider(SsmApi, ABC):
+    def get_parameters(
```

Comment:
Only reordering of methods here (moving util methods to the bottom), no functional changes.